### DISCUSSION TITLE
DuckDB CLI Autocomplete

### DISCUSSION THREAD

**Discussion by cpaika** [duckdb/duckdb#9924] [Ideas]
Similar to modern shell's like `zsh`, it would be great quality of life improvement if in the DuckDB CLI there was an autocomplete.  Like you type part of a command `CAL`, press up arrow, and it pages through commands with that prefix you've run in the past like `CALL load_aws_credentials()` in this example.

**Comment by Tishj** [comment]
https://duckdb.org/docs/extensions/autocomplete

Triggers with tab in the CLI 👍

**Comment by cpaika** [comment]
Ah missed this, thanks!
